l1-contracts/contracts/bridge/L1NativeTokenVault.sol

```diff
@@ -224,23 +198,35 @@ contract L1NativeTokenVault is IL1NativeTokenVault, IL1AssetHandler, Ownable2Ste
         bytes32 _assetId,
         bytes calldata _data
     ) external payable override onlyBridge whenNotPaused {
-        (uint256 _amount, address _depositSender) = abi.decode(_data, (uint256, address));
+        // slither-disable-next-line unused-return
+        _mintOrRecoverInternal(_chainId, _assetId, _data);
+    }
+
+    function _mintOrRecoverInternal(
```

Comment:
_mintTokensOrRecoverTransfer 
or 
_mintOrRecover 
sound better to me, as the underscore suggests it's internal